e2e/testcafe-devextreme/tests/chat/errorList.ts

```diff
@@ -8,7 +9,7 @@ import { getFullThemeName, testScreenshot } from '../../helpers/themeUtils';
 fixture.disablePageReloads`ChatErrorList`
   .page(url(__dirname, '../container.html'));
 
-test('Errorlist appearance', async (t) => {
+test.clientScripts([{ content: 'MockDate.set(\'2024-10-18\')' }])('Errorlist appearance', async (t) => {
```

Comment:
Let's move it to client function instead of scripts: `await (ClientFunction(() => { MockDate.set('2024-10-18') }))()`